examples/tests/copy.earth

```diff
@@ -0,0 +1,167 @@
+
+FROM alpine:3.11
+WORKDIR /test
+
+all:
+    BUILD +copy1
+    BUILD +copy2
+    BUILD +copy3
+    BUILD +copy4
+    BUILD +copy5
+    BUILD +copy6
+    BUILD +copy-art1
+    BUILD +copy-art2
+    BUILD +copy-art3
+    BUILD +copy-art4
+    BUILD +copy-art5
+    BUILD +copy-art6
+
+artifact:
+    COPY --dir in in
+    SAVE ARTIFACT in
+
+copy1:
+    COPY --dir in copied
+    RUN find copied | sort | tee ./actual
+    RUN echo "copied
+copied/root
+copied/sub
+copied/sub/1
+copied/sub/1/file
+copied/sub/2
+copied/sub/2/file
+copied/sub/file" >./expected
+    RUN diff -b ./actual ./expected
+
+copy2:
+    COPY --dir in copied/
+    RUN find copied | sort | tee ./actual
+    RUN echo "copied
+copied/in
+copied/in/root
+copied/in/sub
+copied/in/sub/1
+copied/in/sub/1/file
+copied/in/sub/2
+copied/in/sub/2/file
+copied/in/sub/file" >./expected
```

Comment:
If you use `-printf '%P\n' as a find option it omits the leading directory and clarifies it some IMO`

Reply:
Seems that `-printf` is not available in the `find` available in `alpine` for some reason :-(